Scripts/timer_canvas.gd

```diff
@@ -1,9 +1,24 @@
 extends CanvasLayer
 
 @onready var timer_text: Label = $timer_text
-@export var timer: float = 120.0
+@export var time_limit: float = 120.0
+var timer: float
```

Comment:
timer is vague, be a little more specific if you can time_remaining for example

also timer, overlaps with an existing object and function so should be avoided.